packages/aws-workers/README.md

```diff
@@ -0,0 +1,43 @@
+# KubeZero AWS Workers Package
```

Comment:
I'd say we don't need this readme for each provider, so we can add a generic instructions under `packages/README.md`